python/tvm/relax/expr.py

```diff
@@ -99,7 +99,7 @@ def _binary_op_helper(lhs: "ExprWithOp", rhs: "ExprWithOp", op: Callable) -> "Ex
     if not isinstance(lhs, Expr):  # type: ignore
         raise ValueError("lhs must be Expr")
     if isinstance(rhs, Expr):  # type: ignore
-        return op(lhs, rhs)
+        return op(lhs, rhs, rhs.span)
```

Comment:
Should there be some sort of unification between the lhs and rhs spans? Or is the thought that they would be the same so we just pick one arbitrarily?

Reply:
I've added a unify_span function that gets called here instead